jest.config.js

```diff
@@ -1,5 +1,27 @@
-const { compilerOptions } = require('./tsconfig.json');
+const tsConfig = require('get-tsconfig').getTsconfig();
+const compilerOptions = tsConfig.config.compilerOptions;
+const { baseUrl, paths } = compilerOptions;
+const useSwc = !tsConfig.path.includes('packages/cli');
 
+/** @type {import('@swc/types').Config} */
+const swcJestOptions = {
+	jsc: {
+		parser: {
+			syntax: 'typescript',
+			tsx: false,
+			decorators: true,
+		},
+		transform: {
+			legacyDecorator: true,
+			decoratorMetadata: true,
```

Comment:
Can you clarify what this option is for? The [docs explanation](https://swc.rs/docs/configuration/compilation#jsctransformdecoratormetadata) would seem to suggest it's to support decorator metadata, but if decorator metadata is unsupported, I'm unclear on what this actually does then.